estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,12 @@
+from odoo import fields, models
+
+class EstatePropertyOffer(models.Model):
+    _name="estate.property.offer"
+    _description="Estate Property Offer Model"
+
+    price= fields.Float()
```

Comment:
```suggestion
    price = fields.Float()
```
it should be symetrical add single space on both sides of the field definition 
update everywhere.